services/brig/package.yaml

```diff
@@ -47,6 +47,7 @@ library:
   - Brig.Options
   - Brig.Provider.DB
   - Brig.RPC
+  - Brig.Run
```

Comment:
i probably would have removed the `exposed modules` section, but this is also fine.

Reply:
I was wondering about that; wasn't sure whether it was a pattern or just something left over; in certain cases it might be nice to know what's explicitly accessible; but maybe not? What do you think?